psi_cardinality/c/psi_cardinality_client_c.cpp

```diff
@@ -0,0 +1,73 @@
+#include "psi_cardinality_client_c.h"
+
+#include "internal_utils.h"
+#include "psi_cardinality_client.h"
```

Comment:
Please use the full path here, i.e., `psi_cardinality/cpp/psi_cardinality_client.h`. IDEs sometimes have problems tracking includes with relative paths that are not in the same directory, and we might want to rename the files in the future.